src/main/java/uk/gov/ons/ctp/response/action/representation/ActionRulePostRequestDTO.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.ons.ctp.response.action.representation;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.util.UUID;
+
+/**
+ * Domain model object for representation.
+ */
+@Data
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PUBLIC)
+public class ActionRulePostRequestDTO {
+
+  @NotNull
+  private UUID actionPlanId;
+
+  @NotNull
+  @Size(max = 100)
+  private String actionTypeName;
+
+  @NotNull
+  @Size(max = 100)
+  private String name;
+
+  @NotNull
+  @Size(max = 250)
+  private String description;
+
+  @NotNull
+  private Integer daysOffset;
+
+  @Min(value = 1)
+  @Max(value = 5)
+  private Integer priority;
```

Comment:
I've only seen 3's in the database?

Reply:
The api.md for action priority had 1-5, so I assumed it's the same for action plan. I will double check this though.

Reply:
Found this: COMMENT ON COLUMN actionrule.priority IS '1 = highest, 5 = lowest';
in the initial schema also that column is set to default to 3 which might explains why everything is 3.